jobs/config.json

```diff
@@ -8451,6 +8451,20 @@
       "sig-big-data"
     ]
   },
+  "ci-kubernetes-e2e-node-canary": {
+    "args": [
+      "--up=false",
+      "--down=false",
+      "--gcp-zone=us-central1-f",
+      "--test_args=--nodes=8 --skip=\"\\[Flaky\\]|\\[Serial\\]\"",
+      "--node_args=--images=cos-stable-60-9592-76-0 --image-project=cos-cloud --kubelet-flags=\"--cgroups-per-qos=true --cgroup-root=/\"",
+      "--timeout=60m"
+    ],
+    "scenario": "kubernetes_e2e",
```

Comment:
I think node e2e is part of e2e, if the entry point will be from kubetest, it can be covered by e2e scenario then.